tendrl/AlertNotificationManager.py

```diff
@@ -0,0 +1,28 @@
+import tendrl.utils as utils
```

Comment:
All the sources related to notifications could be under tendrl/notifications I feel. This is would be inline with tendrl/gluster_bridge